frontend/src/components/panels/project_filters_panel/filter_map_components/CheckBox.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import CheckBoxRow from '../filter_map_components/CheckBoxRow';
+
+class CheckBox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.toggleView = this.toggleView.bind(this);
+        this.state = { checked: false };
```

Comment:
I guess I had originally envisioned storing the state of all three CheckBoxRow components in one CheckBox when I suggested lifting state up to a parent component. After thinking about it, it seems like ProjectFiltersPanel is the natural component to store state and interact with the api, so this can probably stay as it is. Sorry for any confusion I caused.

Reply:
@galbwe so just to be clear, the mapping of one CheckBox to CheckBowRow is fine as is?

Reply:
@galbwe so just to be clear, the mapping of one CheckBox to CheckBowRow is fine as is?

Reply:
@clementkng yeah I'd say leave it for now. We should probably open an issue for handling the state of the filter components.